vassal-app/src/main/java/VASSAL/build/BadDataReport.java

```diff
@@ -37,6 +37,7 @@
   private String message;
   private String data;
   private Throwable cause;
+  private boolean reportable = true;
```

Comment:
Just a thought: These could all be final, and you wouldn't need `setReportable()`, right? Then `BadDataReport` could be an immutable object.